packages/discord.js/typings/index.d.ts

```diff
@@ -4700,6 +4711,8 @@ export interface BaseApplicationCommandData {
   dmPermission?: boolean;
   defaultMemberPermissions?: PermissionResolvable | null;
   nsfw?: boolean;
+  contexts?: readonly InteractionContextType[];
+  integration_types?: readonly ApplicationIntegrationType[];
```

Comment:
```suggestion
  integrationTypes?: readonly ApplicationIntegrationType[];
```
every other key in the interface uses camelcase